central/role/sachelper/cluster.go

```diff
@@ -0,0 +1,81 @@
+package sachelper
+
+import (
+	"context"
+
+	clusterMappings "github.com/stackrox/rox/central/cluster/index/mappings"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/postgres/schema"
+	"github.com/stackrox/rox/pkg/sac/effectiveaccessscope"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/set"
+)
+
+////////////////////////////////////////////////////////////////////////////////
+// Helpers                                                                    //
+//                                                                            //
```

Comment:
Let's remove this, I don't think it's worthwhile having this header in this or any of the other files (i.e. `cluster.go, extract.go...`).